frontend/pages/register/_flow/team.vue

```diff
@@ -328,7 +328,8 @@ export default class extends Vue {
       }
 
       data.skills = result.data?.skills;
-      data.topics = result.data?.teamNames;
+      data.topics = result.data?.teamNames.sort(); // sort topics lexicographically
```

Comment:
Please don't do that in the frontend. Sort the data in the backend. Such, we can influence what is displayed how better.